exercises12/main.js

```diff
@@ -0,0 +1,67 @@
+// exercise 1
+function isEqual (arr1, arr2) {
+  if (arr1.length === arr2.length) {
+    for (let i = 0; i < arr1.length; i ++) {
+      if (arr1[i] !== arr2[i]) {
+        return false
+      }
+    }
+    return true
+  }
+  return false
```

Comment:
Đổi ngược điều kiện đi, sẽ dễ nhìn hơn
```suggestion
  if (arr1.length !== arr2.length) {
    return false
  }
 
  for (let i = 0; i < arr1.length; i ++) {
    if (arr1[i] !== arr2[i]) {
      return false
    }
  }
  return true
```